pyflakes/checker.py

```diff
@@ -1139,15 +1146,15 @@ def IMPORTFROM(self, node):
                 # Only Python 2, local import * is a SyntaxWarning
                 if not PY2 and not isinstance(self.scope, ModuleScope):
                     self.report(messages.ImportStarNotPermitted,
```

Comment:
Are you against special casing `from . import *`? It seems like the only special case we might need and should be simple enough, right? (i.e., `module` above would just be `.` but it looks like that should be caught by the `if module.endswith('.')` so maybe I'm looking at the wrong place?)